components/11-argo-workflow/kustomization.yaml

```diff
@@ -0,0 +1,18 @@
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+  - base
+
+helmGlobals:
+  chartHome: ../../charts/
+
+helmCharts:
+  - name: argo-workflows
+    includeCRDs: true
+    namespace: argo
+    valuesFile: values.yaml
+    releaseName: argo-workflow
+    version: 0.40.10
+    repo: https://argoproj.github.io/argo-helm
```

Comment:
Given what the project says about the charts being community maintained and not officially maintained. The fact that the kustomize native config is the officially maintained config. We also want to install this in the namespaced form and not cluster wide which the charts don't support. I think we switch.